test/test_va_api_query_imageformat.cpp

```diff
@@ -0,0 +1,66 @@
+ /*
+ * Copyright (C) 2017 Intel Corporation. All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sub license, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice (including the
+ * next paragraph) shall be included in all copies or substantial portions
+ * of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT.
+ * IN NO EVENT SHALL PRECISION INSIGHT AND/OR ITS SUPPLIERS BE LIABLE FOR
+ * ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+#include "test_va_api_query_imageformat.h"
+#include<functional>
+
+namespace VAAPI {
+VAAPIQueryImageFormat::VAAPIQueryImageFormat()
+{
+    m_vaDisplay = doInitialize();
+}
+
+VAAPIQueryImageFormat::~VAAPIQueryImageFormat()
+{
+    doTerminate();
+}
+
+TEST_P(VAAPIQueryImageFormat, DriverSupportedImageFormats)
+{
+    uint32_t currentFormat = GetParam();
+    uint32_t result;
+
+    doGetMaxNumImageFormats();  
+ 
+    doQueryImageFormats();
+
+    result = doFindImageFormatInList(currentFormat);
+    if(result == currentFormat)
+	SUCCEED();
+    else if (!result)
+	{
```

Comment:
This brace should be in line with the 'else if' for clarity. Please modify the rest of the else if to follow standard indention rules. The '}' on line 57 should be moved to match this '{'.

Reply:
Fixed.